mmv1/products/compute/RouterNat.yaml

```diff
@@ -481,7 +482,7 @@ properties:
       The network tier to use when automatically reserving NAT IP addresses.
       Must be one of: PREMIUM, STANDARD. If not specified, then the current
       project-level default tier is used.
-    default_from_api: true
+    default_value: "STANDARD"
```

Comment:
Looks like this change introduced a breaking-change, is this the intended behaviour?